uportal-war/src/main/webapp/media/skins/respondr/common/common_skin.xml

```diff
@@ -23,6 +23,14 @@
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
+	<!-- Data Table Bootstrap Includes -->
+    <css included="plain" resource="true">/rs/datatables/1.9.4/plugins/integration/bootstrap/3/dataTables.bootstrap.css</css>
+    <css included="aggregated" resource="true">/rs/datatables/1.9.4/plugins/integration/bootstrap/3/dataTables.bootstrap.css</css>
+    
+	<!-- Data Table ColumnFilterWidgets Includes -->    <!-- Data Table ColumnFilterWidgets Includes -->
+	<css included="plain" resource="true">/rs/datatables/1.9.4/extras/column-filter-widgets/css/ColumnFilterWidgets.css</css>
+    <css included="aggregated" resource="true">/rs/datatables/1.9.4/extras/column-filter-widgets/css/ColumnFilterWidgets.css</css>
+        
```

Comment:
Anthony is correct:

https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd

<xs:element name="resources"><xs:complexType><xs:sequence><xs:element name="parameter" type="parameter" minOccurs="0" maxOccurs="unbounded"/><xs:element name="css" type="css" minOccurs="0" maxOccurs="unbounded"/><xs:element name="js" type="js" minOccurs="0" maxOccurs="unbounded"/></xs:sequence></xs:complexType></xs:element>